services/web-app/components/nav-tree/nav-tree.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { unstable_TreeNode as TreeNode, unstable_TreeView as TreeView } from '@carbon/react'
+import { useRouter } from 'next/router'
+import PropTypes from 'prop-types'
+
+import styles from './nav-tree.module.scss'
+
+const NavTree = ({ activeItem, items = [], label }) => {
+  const router = useRouter()
+  const toggle = function () {
+    return null
+  }
+
```

Comment:
An option for reducing the duplication here would be to recursively generate the hierarchy of TreeNodes. That would enable the max depth to be configurable as an added bonus :)